scripts/components/OpenSearch/bwctest.sh

```diff
@@ -0,0 +1 @@
+# TODO: This is used by ScriptFinder tests, needs a working bwctest.sh for OpenSearch.
```

Comment:
Do we need to have a header for this file as well?

Reply:
Added.